suggestions/views.py

```diff
@@ -92,7 +93,22 @@ def get_context_data(self, **kwargs: Any):
             raise PermissionDenied("Please log in.")
         if not (self.request.user.is_staff or self.request.user == self.object.user):
             raise PermissionDenied("Logged-in user cannot view this suggestion")
+        
+        context["pk"] = self.kwargs.get("pk", None)
         return context
+        return context
```

Comment:
duplicated `return context`, the linter should probably catch unreachable statements like this, tbh